app/Libraries/UKCP.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Libraries;
+
+class UKCP
+{
+
+    /** @var string */
+    private $apiKey;
+
+    /**
+     * UKCP constructor.
+     */
+    public function __construct()
+    {
+        $this->apiKey = env('UKCP_KEY');
```

Comment:
Since we cache the env variable in prod (I assume), should we extract this to a config file: REF: https://laravel.com/docs/5.7/configuration#configuration-caching

Reply:
1f67400